corehq/apps/case_importer/views.py

```diff
@@ -32,6 +34,14 @@ def render_error(request, domain, message):
     return HttpResponseRedirect(base.ImportCases.get_url(domain=domain))
 
 
+def validate_column_names(column_names, invalid_column_names):
+    for column_name in column_names:
+        try:
+            lxml.etree.fromstring('<{name}></{name}>'.format(name=column_name))
+        except XMLSyntaxError:
```

Comment:
@snopoke do you still think we should use `lxml.etree.Element('name&place')` here?

Reply:
Yes